### ISSUE TITLE
Unable to mount ESP

### ISSUE THREAD

**Issue by mathewmarcus** [grazzolini/mkinitcpio-chkcryptoboot#10]
Currently, when mkinitcpio tries to run the `mount $ESP` command in the hook, it produces the following error:
```
mount: /boot/efi: can't find UUID=XXXX-XXXX
sha512sum: /boot/efi/EFI/grub/grubx64.efi: FAILED open or read
sha512sum: WARNING: 1 listed file could not be read

CHKCRYPTOBOOT ALERT!

CHANGES HAVE BEEN DETECTED IN YOUR BOOT LOADER EFISTUB!

Your kernel cmdline contain the correct parameters.
YOU ARE STRONGLY ADVISED NOT TO ENTER YOUR ROOT CONTAINER PASSWORD!
Please type uppercase yes to continue:
```

This is the case even when run for the first time. Upon closer inspection, by adding `echo` statements to the chkcryptoboot_hook file, it appears that no block devices can be found. However, additional `echo` statements in the encrypt hook, which immediately follows this hook, revealed that at that point the block devices were recognized. 

After inspecting the encrypt hook code, it appears that it is using the [resolve_device](https://git.archlinux.org/mkinitcpio.git/tree/init_functions#n325) function to ensure that all block devices are available. Calling this function from the chkcryptoboot_hook ensures that all required devices and partitions are available.

**Comment by ncoder-1** [comment]
Getting the same error, can you build a new release and update the AUR?

**Comment by grazzolini** [comment]
Fixed with #16
